packages/components/src/components/tables/demos/table-grouped.demo.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, h, State } from '@stencil/core';
+import { router } from '@eventstore-ui/router';
+import type {
+    ClickRowEvent,
+    ClickSortEvent,
+    TableCells,
+    TableSort,
+} from '../types';
+
+interface DummyData {
+    name: string;
+    value: string;
+    amount: number;
+}
+
+function alphabet(num: number) {
+    let s = '';
+    let t;
+    while (num > 0) {
+        t = (num - 1) % 26;
+        s = String.fromCharCode(65 + t) + s;
+        num = ((num - t) / 26) | 0;
+    }
+    return s;
+}
+
+/** Header grouping es-table demo. */
+@Component({
+    tag: 'es-table-grouped-demo',
+    styleUrl: './table-basic.css',
+    shadow: true,
+})
+export class Demo {
+    private data: Map<string, DummyData> = new Map(
+        Array.from({ length: 200 }, (_, i) => [
+            alphabet(i + 1),
+            {
+                name: alphabet(i + 1),
+                value: 'Something here',
+                amount: Math.round(Math.random() * 20),
+            },
+        ]),
+    );
+
+    @State() keys: string[] = Array.from(this.data.keys());
+    @State() sort: TableSort = ['name', 'ascending'];
+    @State() active?: string;
+
+    componentWillLoad() {
+        router.init();
+        this.applySort(this.sort);
+    }
+
+    render() {
+        return (
+            <es-table
+                stickyHeader
+                cells={this.cells}
+                rows={this.keys}
+                getCellData={(d) => this.getCellData(d)}
+                rowClass={this.rowClass}
+                sort={this.sort}
+                onClickRow={this.onClickRow}
+                onClickSort={this.onClickSort}
+                columns={[
+                    'name',
+                    'type',
+                    'sent_rate',
+                    'sent_current',
+                    'sent_pending',
+                    'lorem',
+                    'sent_rate',
+                    'sent_current',
+                    'sent_pending',
+                    'lorem',
+                ]}
+            />
+        );
+    }
+
+    private onClickRow = (e: ClickRowEvent<DummyData>) => {
+        const { name } = e.detail.data;
+        this.active = name;
+    };
+
+    private rowClass = (row: DummyData) => ({
+        selectable: true,
+        selected: row.name === this.active,
+    });
+
+    private onClickSort = (e: ClickSortEvent) => {
+        const row = e.detail;
```

Comment:
```diff
- const row = e.detail;
+ const column = e.detail;
```
`e.details` returns a column name, having the variable named `column` makes more sense